test/hotspot/jtreg/gc/shenandoah/generational/TestOldGrowthTriggers.java

```diff
@@ -58,14 +58,6 @@ public static void makeOldAllocations() {
                 int replaceIndex = r.nextInt(ArraySize);
                 int deriveIndex = r.nextInt(ArraySize);
                 switch (i & 0x3) {
```

Comment:
...otherwise could you just do (i & 0x1) here and change cases 2 & 3 to cases 0 & 1?